### PR TITLE
[sc-13244] Add `Workspace` and `Object` entities

### PR DESCRIPTION
**Pull request by richardxgao**
Add the following new entities:

- Object
- Workspace

An object is a member of a workspace.

Create an `ObjectMember` relationships between an object and a workspace if their `workspaceId` fields match.
There is no need to create an intermediate `ObjectMember` node because an object already has a unique ID (`globalId`) which is unique for a given workspace and object.

### CODE REVIEW

**Review by shash-sgnl — DISMISSED** [review]

**Review by shash-sgnl — APPROVED** [review]

File: jira.sgnl.yaml [hunk 1]
```diff
@@ -383,6 +384,112 @@ entities:
...
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: workspaceId
+        externalId: workspaceId
+        type: String
+        indexed: true
+        uniqueId: true
+
+  # WARNING: In order to sync Objects, you must specify the objectsQlQuery in the adapterConfig.
```
Comment (shash-sgnl) [inline jira.sgnl.yaml:405]: @richardxgao can we default the adapterConfig to a query that will return 200 instead?
The goal of templates is to have a working one for each SoR. So, any user must be able to just use the template as is and start ingesting data.

File: jira.sgnl.yaml [hunk 2]
```diff
@@ -383,6 +384,112 @@ entities:
...
+      - name: created
+        externalId: created
+        type: DateTime
+      - name: updated
+        externalId: updated
+        type: DateTime
+      - name: timestamp
+        externalId: timestamp
+        type: Int64
+      # TODO: We currently don't allow attributes to be start with a non-alphanumeric char.
```
Comment (shash-sgnl) [inline jira.sgnl.yaml:485]: Please remove the attribute if it's not supported.

**Review by shash-sgnl — CHANGES_REQUESTED** [review]

File: jira.sgnl.yaml (hunk 2 above)
Comment (richardxgao) [reply jira.sgnl.yaml:485]: Removed.

File: jira.sgnl.yaml (hunk 1 above)
Comment (richardxgao) [reply jira.sgnl.yaml:405]: From offline discussions, we've decided to

1) Default the `adapterConfig` to be something like:
```json
{
    "objectsQlQuery": "objectType = INPUT_REQUIRED"
}
```
which will return 200 when syncing `Objects`.

2) Note in the documentation that the customer should change this in the config.

Comment (richardxgao) [reply jira.sgnl.yaml:405]: @shash-sgnl Maybe we should also note that this config field is only used for `Object` entities, so if they don't care about it they can just leave the field as-is, or not sync the entity at all.

**Review by shash-sgnl — APPROVED** [review]

**Comment by shash-sgnl** [comment]
@richardxgao looks good. merged this.

**Pull request merged**